graphql_compiler/tests/snapshot_tests/test_orientdb_match_query.py

```diff
@@ -45,10 +46,20 @@ def execute_graphql(
     if test_data.type_equivalence_hints:
         # For test convenience, we accept the type equivalence hints in string form.
         # Here, we convert them to the required GraphQL types.
-        schema_based_type_equivalence_hints = {
-            schema.get_type(key): schema.get_type(value)
-            for key, value in six.iteritems(test_data.type_equivalence_hints)
-        }
+        schema_based_type_equivalence_hints = {}
```

Comment:
To make `mypy` happy, you might need to declare the type here: `Dict[Union[GraphQLInterfaceType, GraphQLObjectType], GraphQLUnionType]`